kde-plasma/powerdevil/powerdevil-6.4.5-r1.ebuild

```diff
@@ -58,6 +58,7 @@ RDEPEND="${COMMON_DEPEND}
 	!<kde-plasma/plasma-workspace-6.1.90:*
 	>=dev-qt/qtdeclarative-${QTMIN}:6
 	|| (
+		sys-apps/tuned[ppd]
```

Comment:
This changes the default (at least for ~amd64 for now) when none of the 3 alternatives are present on emerge powerdevil. Could you add the rationale why we prefer this now in git summary, please?